README.md

```diff
@@ -122,6 +126,25 @@ The official documentation of the upstream layer can be found [here](https://git
 
     For more information about available context propagators, see the [Propagator settings](https://github.com/signalfx/splunk-otel-python/blob/main/docs/advanced-config.md#trace-propagation-configuration) for the Splunk distribution of OpenTelemetry Python.
 
+### Node.js wrapper
+
+The Node.js wrapper is based on Splunk Distribution of OpenTelemetry for Node.js version `0.14.0`. 
```

Comment:
Are you sure you want to hard-code versions? 😄

Reply:
Yep, each version change of dependencies will require a PR anyway.